templates/confirmation-email-sent.html

```diff
@@ -21,4 +21,13 @@ <h1 data-qa="confirmation-text">{{ _("A confirmation email has been sent to {ema
 
   {% include 'partials/sign-out-pre-footer-panel.html' %}
 
+  {% call onsPanel({
```

Comment:
The design has it before "Didn't receive an email?"

Reply:
Done 👍